swagger.yaml

```diff
@@ -3584,7 +3584,7 @@ paths:
           name: platform_type
           description: The provider platform type.
           type: string
-          enum: [ 'baremetal', 'none', 'nutanix', 'vsphere', 'oci' ]
+          enum: [ 'baremetal', 'none', 'nutanix', 'vsphere', 'oci', 'external' ]
```

Comment:
Let's remove oci

Reply:
don't we want to remove it once the UI has migrated to external+platform name?

Reply:
Yes